pkg/cmd/server/api/v1/conversions.go

```diff
@@ -99,6 +99,16 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 				// The final value of OAuthConfig.MasterCA should never be nil
 				obj.OAuthConfig.MasterCA = &s
 			}
+
+			if len(obj.ImagePolicyConfig.AllowedRegistriesForImport) == 0 {
+				obj.ImagePolicyConfig.AllowedRegistriesForImport = []string{
```

Comment:
fixed. also I moved the default whitelist to types.go (instead of burrying it in master_args)